### PR TITLE
Remove usage of "filters" in OWNERS

### PR DESCRIPTION
**Pull request by xpivarc**
### What this PR does
The documentation states:
"WARNING: The approve plugin does not currently respect filters. Until that is fixed, filters should only be used for the labels key (as shown in the above example)"

From now on we can only use OWNERS files per directory.

Fixes #

Links to places where the discussion took place: 

### Release note
```release-note
NONE
```

### CODE REVIEW

**Review by jean-edouard — APPROVED** [review]

**Comment by xpivarc** [comment]
/hold
Forgot about the cluster-up...

**Comment by xpivarc** [comment]
I need to address `cluster-up` in `kubevirtci`

**Review by brianmcarey — COMMENTED** [review]
/lgtm

**Review by EdDev — COMMENTED** [review]
As this is effecting existing “focused” approvers, I think they need to be notified and asked to explicitly ack this for awareness and chance to seek mitigation.

What I do not understand is why it worked in 99% of the case. I witness only one incident when it did not work well.
Do we understand it?

**Comment by xpivarc** [comment]
> As this is effecting existing “focused” approvers, I think they need to be notified and asked to explicitly ack this for awareness and chance to seek mitigation.
> 
> What I do not understand is why it worked in 99% of the case. I witness only one incident when it did not work well. Do we understand it?

We default here to be safe and use what we know works. I will tag appropriate people to let them now that this did not work predictably and we will try to get back to the approach.

File: OWNERS
Comment (xpivarc) [reply OWNERS:26]: For now feel free to ping me for any approval you need I will raise your lgtm. The folder would work.

**Comment by xpivarc** [comment]
@rthallisey @sradco @machadovilaca @dhiller @brianmcarey 
Notifying you as you will lose approve rights for part of the code base, we will work on reestablishing this.

**Comment by dhiller** [comment]
@xpivarc maybe I am overlooking something, but the filters removal also removed the application of labels for the matching files. Do we have something to reestablish this?

**Comment by xpivarc** [comment]
> @xpivarc maybe I am overlooking something, but the filters removal also removed the application of labels for the matching files. Do we have something to reestablish this?

Good catch, I think we can revert that part. I am still not confident about the approve+filter behavior tho

**Pull request merged**
